src/keybindings.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+import curses
+
+
+# ASCII number of the command.
+CTRL_V = 22
+ESC = 27
+
+ACTION_TABLE = {
```

Comment:
Some nits
- Change **action** wording to **command** (just suggestion)
- Declare a enum class right here (see `SelectionMode` in **pick.py** for e.g.) and create enums for the actions instead of using raw strings
- Invert this mapping and use lists as values, such as `Command.down: [curses.KEY_DOWN, ord('j')]`  
Then you can post-process the map to invert it and easily lookup just like you are currently doing in `action_for_ordinal`